cmake/BLTMacros.cmake

```diff
@@ -302,7 +303,7 @@ endmacro(blt_add_target_link_flags)
 ##------------------------------------------------------------------------------
 macro(blt_register_library)
 
-    set(singleValueArgs NAME TREAT_INCLUDES_AS_SYSTEM)
+    set(singleValueArgs NAME OBJECT TREAT_INCLUDES_AS_SYSTEM)
```

Comment:
Please document ``OBJECT``, as well as its default value in the signature for ``blt_register_library`` above (around line 270)

Reply:
Thanks for the updated docs -- it wasn't obvious (to me) that this should not be manually set by users (but it makes sense).